db/migrations/20200211143227_addPlaylistFavorites.js

```diff
@@ -1,7 +1,7 @@
 
 exports.up = function(knex) {
   return Promise.all([
-    knex.schema.createTable('playlistFavorites', function(table) {
+    knex.schema.createTable('playlist_favorites', function(table) {
```

Comment:
Thanks for troubleshooting the snake vs camelCase on database/knex issue with this